style3.css

```diff
@@ -26,29 +25,33 @@ body{
 
 nav{
 	width:100%;
-	background-color: red;
+	background-color:#0C59CF;
 	overflow:auto;
-	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
 /* Set up position for the text in nav to the right */
 li{
 	float:right;
 }
-
+/* Set up position for the logo */
+.logo img{
+	position:absolute;
+	margin-top:-20px;
+}
 
 /* Set up Drop down menu for */
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
27/09/2021: change the fonts for my link in the nav bar to Open San which was the fonts that I decided to use for normal text. Also I make the text to be bold so It can be clearly read by all user(for both old and young people). Lastly because I make the background color for the nav bar to navy blue so I change the color of those text to white which make it display a great contrast for for the admin (could be old or young). And so I can be consistence because this type of setting was like what in the user nav